geo/Geoserver.py

```diff
@@ -945,14 +945,20 @@ def update_layergroup(
     def delete_layergroup(
         self,
         layergroup_name: str,
+        workspace: Optional[str] = None
     ) -> str:
         try:
-            if self.get_layergroup(layer_name=layergroup_name) is None:
+            try: 
```

Comment:
Why do we need this block of `try`, and `except` blocks? I didn't see any required information inside the `try` block. I think it should remove this block.

Reply:
The idea was to raise the exact error message we had here before. I have removed it now, so you now get whatever error message ist returned by get_layergroup.